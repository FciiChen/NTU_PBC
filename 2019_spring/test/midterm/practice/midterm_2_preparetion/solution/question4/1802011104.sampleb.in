過了一會，各人突然聞到一陣淡淡的花香。玄難叫道：「敵人放毒，快閉住了氣，聞解藥。」但過了一會，不覺有異，反覺頭腦清爽，似乎花香中並無毒質。 
　　外面那人說道：「七姊，是你到了麼？五哥屋中有個怪人，居然自稱安祿山。」一個女子聲音道：「只大哥還沒到。二哥、三哥、四哥、六哥、八弟，大家一齊現身吧！」 
　　她一句話甫畢，大門外突然大放光明，一團奇異的亮光裹著五男一女。光亮中一個黑鬚老者大聲道：「老五，還不給我快滾出來。」他右手中拿著方方的一塊木板。那女子是個中年美婦。其餘四人中兩個是儒生打扮，一人似是個木匠，手持短斧，背負長鋸。另一個卻青面獠牙，紅髮綠須，形狀可怕之極，直是個妖怪，身穿一件亮光閃閃的錦袍。 
　　鄧百川一凝神間，已看出這人是臉上用油彩繪了臉譜，並非真的生有異相，他扮得便如戲台上唱戲的伶人一般，適才既扮唐明皇又扮梅妃的，自然便是此君了，當下朗聲道：「諸位尊姓大名，在下姑蘇慕容氏門下鄧百川。」 
　　對方還沒答話，大廳中一團黑影撲出，刀光閃閃，向那戲子連砍七刀，正是一陣風風波惡。那戲子猝不及防，東躲西避，情勢甚是狼狽。卻聽他唱道：「力拔山兮氣蓋世，時不利兮騅不逝，騅不逝兮可……」但風波惡功勢太急，他第三句沒唱完，便唱不下去了。 
　　那黑鬚老者罵道：「你這漢子忒也無理，一上來便狂砍亂斬，吃我一招『大鐵網』！」手中方板一晃，便向風波惡頭頂砸到。 
　　風波噁心下嘀咕：「我生平大小數百戰，倒沒見過用這樣一塊方板做兵刃的。」單刀疾落，便往板上斬去。錚的一聲響，一刀斬在板緣之上，那板紋絲不動，原來這塊方板形似木板，卻是鋼鐵，只是外面漆上了木紋而已。風波惡立時收刀，又待再發，不料手臂回縮，單刀竟爾收不回來，卻是給鋼板牢牢的吸住了。風波惡大驚，運勁一奪，這才使單刀與鋼板分離，喝道：「邪門之至！你這塊鐵板是吸鐵石做的麼？」 
　　那人笑道：「不敢，不敢！這是老夫的吃飯傢伙。」風波惡一瞥之下，見那板上縱一道、橫一道的畫著許多直線，顯然便是一塊下圍棋用的棋盤，說道：「希奇古怪，我跟你們鬥！」進刀如風，越打越快，只是刀身卻不敢再和對方的吸鐵石棋盤相碰。 
　　那戲子喘了口氣，粗聲唱道：「騅不逝兮可奈何，虞兮虞兮奈若何？」忽然轉作女子聲音，嬌嬌滴滴的說道：「大王不必煩惱，今日垓下之戰雖然不利，賤妾跟著大王，殺出重圍便了。」 
　　包不同喝道：「直娘賤的楚霸王和虞姬，快快自刎，我乃韓信是也。」縱身伸掌，幾那戲子肩頭抓去。那戲子沉肩躲過，唱道：「大風起兮雲飛揚，安得……啊唷，我漢高祖殺了你韓信。」左手在腰間一掏，抖出一條軟鞭，劇的一聲，向包不同抽去。 
　　玄難見這幾人鬥得甚是兒戲，但雙方武功均甚了得，卻不知對方來歷，眉頭微皺，喝道：「諸位暫且罷手，先把話說明白了。」 
　　但要風波惡罷手不鬥，實是千難萬難，他自知身受寒毒之後，體力遠不如平時，而且寒毒隨時會發，甚是危險，一柄單刀使得猶如潑風相似，要及早勝過了對方。 
　　四個人酣戰聲中，大廳中又出來一個，嗆啷啷一聲響，兩柄戒刀相碰，威風凜凜，卻是玄痛。他大聲說道：「你這批下毒害人的奸徒，老和尚今日大開殺戒了。」他連日苦受寒毒的折磨，無氣可出，這時更不多問，雙刀便向兩個儒生砍去。一個儒生閃身避過，另一個探手入懷摸出一枝判官筆模樣的兵刃，施展小巧功夫，和玄痛鬥了起來。另一個儒生搖頭晃腦說道：「奇哉怪也！出家人竟也有這麼大的火氣，卻不知出於何典？」伸到懷中一摸，奇道：「咦，哪裡去了？」左邊袋中摸摸，右邊袋裡掏掏，抖抖袖子，拍拍胸口，說什麼也找不到。 
　　虛竹好心起，問道：「施主，你找什麼？」那儒生道：「這位大和尚武功甚高，我兄弟鬥他不過，我要取出兵刃，來個以二敵一之勢，咦，奇怪，奇怪！我的兵刃卻放到哪裡去了？」敲敲自己額頭，用心思索。虛竹忍不住噗哧一笑，心想：「上陣要打架，卻忘記兵器放在哪裡，倒有趣。」又問：「施主，你用是什麼兵刃？」 
　　那儒生道：「君子先禮後兵，我的第一件兵刃是一部書。」虛竹道：「什麼書？是武功秘訣麼？」那儒生道：「不是，不是。那是一部『論語』。我要以聖人之言來感化對方。」包不同插道：「你是讀書人，連『論語』也背不出，還讀什麼書？」那儒生道：「老兄只知其一，不知其二。說到『論語』、『孟子』、『春秋』、『詩經』，我自然讀得滾瓜爛熟，但對是佛門弟子，只讀佛經，儒家之書未必讀過，我背了出來，他若不知，豈不是無用？定要翻出原書來給他看了，他無可抵賴，難以強辯，這才收效。常言道得好，這叫做『有書為證』。」一面說，一面仍在身上各處東掏西模。 
　　包不同叫道：「小師父快打他！」虛竹道：「待這位施主找到兵器，再動手不遲。」那儒生道：「宋楚戰於泓，楚人渡河未濟，行列未成，正可擊之，而宋襄公曰：『擊之非君子』。小師父此心，宋襄之仁也。」 
　　那工匠模樣的人見玄痛一對戒刀上下翻飛，招數凌厲之極，再拆數招，只怕那使判官筆的書生便性命之憂，當揮斧而前，待要且戰。公冶干呼的一掌，向他拍了過去。公冶干模樣斯文，掌力可著實雄渾，有「江南第二」之稱，當日他與蕭峰比酒比掌力，雖然輸了，蕭峰對他卻好生敬重，可見內幾造詣大是不凡。那工匠側身避過橫斧斫來。 
　　那儒生仍然沒找到他那部「論語」，卻見同伴的一枝判官筆招法散亂，底擋不住玄痛雙刀，便向玄痛道：「喂，大和尚。子曰：『君子無終食之間違仁，造次必於是，顛淵問仁，子曰：『克已復禮為仁。一日克已復禮，天下尋仁焉』。夫子又曰：『非禮勿視，非禮勿聽，非禮勿言，非禮勿動』。你亂揮雙刀，狠霸霸的只想殺人，這等行動，毫不『克已』，那是『非禮』之至了。」 
　　虛竹低聲問身旁的少林僧慧方道：「師叔，這人是不裝傻？」慧方搖頭道：「我也不知道。這次出寺，師父吩咐大家小心，江湖上人心詭詐，什麼鬼花樣都幹得出來。」 
　　那書獃子又向玄痛道：「大和尚，子曰：『仁者必有勇，勇者必有仁。』你勇則勇矣，卻未必有仁，算不得是真正的君子。子曰：『已所不欲，勿報施於人』。人家倘若將你殺了，你當然是很不原意的了。你自己既不願死，卻怎麼去殺人呢？」 
　　玄痛和那書生跳蕩前後，揮刀忽鬥，這書獃子隨著玄痛忽東忽西，時左時右，始終不離分三尺之外，不住勸告，武功顯然不弱。玄痛暗自警惕：「這傢伙如此胡言語，顯是要我分心，一找到我招式中的破綻，立時便乘虛而入。此人武功尚在這個使判官筆的人之上，倒是不可不防。」這麼一來，他以六分精神去防書獃，只以四分功夫攻擊使判官筆的書生。那書情勢登時好轉。 
　　又拆十餘招，玄痛焦躁起來，喝道：「走開！」轉戒刀，挺刀柄向那書可胸口撞去。那書閃身讓開，說道：「我見大師武功高強，我四和弟二人以二敵一，也未必鬥你得過，是以良言相勸於你，還是兩罷戰的為是子曰：『參乎！吾道一以貫之。』曾子曰：『夫子之道，忠恕而已矣。』咱們做人，這『恕道』總是要守的，不可太也橫蠻。」 
　　玄痛大怒，刷的一刀，橫砍過去，罵道：「什麼忠恕之道？仁義道德？你們怎麼在棺材裡放毒藥害人？老衲倘若一個不小心，這時早已圓寂歸西了，還虧你說什麼『已所不欲，勿施於人』？你想不想中毒而死啊？」 
　　那書獃子退開兩步，說道：「奇哉！奇哉！誰在棺材放毒藥了？夫棺材者，盛死屍之物也。子曰：『鯉也死，有棺而無槨。』棺材中放毒藥，豈不是連死屍也毒死了？啊喲，不對死人是早死了的。」 
　　包不同插口道：「非也，非也。你們的棺材裡卻不放死屍而放毒藥，只是想毒死我們這些活人。」那書獃子搖頭晃腦的道：「閣下以小人之心，而度君子之腹矣。此處既無棺材，更無毒藥。」 
　　包不同道：「子曰：『唯女子與小人為難養也。』你是小人。」指著對面那中年美婦道：「她是女子。你們兩個，果然難養得很。孔夫子的話，有錯的嗎？」那書獃子一怔，說道：「『王顧左右而言他。』我這句話，我便置之不理，不加答覆了。」 
　　這書獃與包不同一加對答，玄痛少了顧礙，雙刀又使得緊了，那使判官筆的書生登時大見吃緊。那書獃晃身欺近玄痛身邊說道：「子曰：『人而不仁，如禮何？人而不仁，如樂何？』大和尚『人而不仁』，當真差勁之至了。」 
　　玄痛怒道：「我是釋家，你喧腐儒講什麼詩書禮樂，人而不仁，根本打不動我的心。」 
　　那書獃伸起手指，連敲自己額頭，說道：「是極，是極！我這人可說是讀書而呆矣，真正書獃子矣。大和尚明明是佛門子弟，我跟你說孔孟的仁義道德，自然格格不人焉。」 
　　風波久鬥那使鐵製棋盤之人，難以獲勝，時刻稍久，小腹中隱隱感到寒毒侵襲。包不同和那戲子相差別，察覺對方武也不甚高，只是招數變化極繁，一時扮演西施，吐言鶯聲嚦嚦，而且蹙眉捧心，蓮步姍姍，宛然是個絕代佳人的神態，頃刻之間，卻又扮演起酒風流的李太白來，醉態可掬，腳步東倒西歪。妙在他扮演各式人物，均有套武功與配合，手中軟鞭或作美人之長袖，或為文土這采筆，倒令包不同啼筆皆非，一時也奈何他不得。 
　　那書獃自艾了一陣，突然長聲吟道：「既已捨染樂，心得善攝不，若得不馳散，深入相不？」玄難與玄痛都是一驚：「這書獃子當真淵博，連東晉高僧鳩摩羅什的偈句也背得出。」只聽他繼續吟道：「畢竟空相中，其心無所樂，若悅禪智慧，是法性無照。虛誑等無實，亦非停心處。大和尚，下面兩句是什麼？我倒忘記了。」玄痛道：「仁者所得法，幸願示其要。」 
　　那書獃哈哈大笑，道：「照也！照也！你佛家大師，豈不也說『仁者』？天下的道理，都是一樣的。我勸你還是回頭是岸，放下屠刀罷！」 
　　玄痛心中一驚，陡然間大徹大悟，說道：「善哉！善哉！善哉！南無阿彌陀佛，南夫阿彌陀佛。」嗆啷啷兩聲響，兩柄戒刀擲在地下，盤漆而坐，臉露微笑，閉目不語。 
　　那書生和他鬥得甚酣，突然間，見到他這等模樣，倒吃了一驚，手中判官筆並不攻上。 
　　虛竹叫道：「師叔祖，寒毒又發了嗎？」伸的待要相扶，玄難喝道：「別動！」一探玄痛的鼻息，只覺呼吸已停，竟爾圓寂了。玄難雙手合什，念起「往生咒」來。眾少林僧見玄痛圓寂，齊聲大哭，抄起禪杖戒刀，要和兩個書生拚命。玄難說道：「住手！玄痛師弟參悟真如，往生極樂，乃是成了正果，爾輩須得歡喜才是。」 
　　正自激鬥的眾人突然見此變故，一齊罷手躍開。 
　　那書獃大叫：「老五，薛五弟，快快出有人給我一句話激死了，快出來救命！你這他媽的薛神醫再不出來救命，那可乖乖不得了啊！」鄧百川道：「薛神醫不在家中，這位先生……」那書獃仍是放開了嗓門，慌慌張張的大叫：「薛慕華，薛老五，閻王敵，薛神醫，快快滾出來救人哪！你三哥激死人了，人家可要跟咱們過不去啦。」 
　　包不同怒道：「你害死了人，還在假惺惺的裝腔作勢。」呼的一掌，向他拍了過去，左手跟著從右掌掌底穿出，一招「老龍探珠」，逕自抓了的鬍子。那書獃閃身避過。風波惡、公冶乾等鬥得興起，不願便此停手，又打了起來。 
BREAK