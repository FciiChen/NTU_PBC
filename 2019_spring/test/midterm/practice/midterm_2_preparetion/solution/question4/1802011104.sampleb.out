那_獃 11
包_同 10
那_生 8
風_惡 8
判_筆 7
書_子 7
那_子 7
大_尚 6
人_不 5
儒_道 5
